From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Baruch Siach <baruch@tkos.co.il>
Date: Wed, 22 Jul 2015 07:04:33 +0300
Subject: [PATCH] Avoid searching host library path

The $(libdir) variable points to the location of the directory on the target
system, /usr/lib by default. When cross compiling this directory contains the
host libraries which may be different than target libraries. Don't use
$(libdir) in the library search path.

Signed-off-by: Baruch Siach <baruch@tkos.co.il>
Signed-off-by: Joel Stanley <joel@jms.id.au>
[Fabrice: refresh patch for 2.0.32]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://sourceforge.net/p/openipmi/patches/35]
---
 cmdlang/Makefile.am | 3 +--
 unix/Makefile.am    | 6 ++----
 2 files changed, 3 insertions(+), 6 deletions(-)

diff --git a/cmdlang/Makefile.am b/cmdlang/Makefile.am
index 264ee12be79f..0b3843784656 100644
--- a/cmdlang/Makefile.am
+++ b/cmdlang/Makefile.am
@@ -15,7 +15,7 @@ libOpenIPMIcmdlang_la_SOURCES = cmdlang.c cmd_domain.c cmd_entity.c cmd_mc.c \
 	$(top_builddir)/utils/libOpenIPMIutils.la \
 	$(top_builddir)/lib/libOpenIPMI.la
 libOpenIPMIcmdlang_la_LDFLAGS = -rdynamic -version-info $(LD_VERSION) \
-	-L$(libdir) -no-undefined
+	-no-undefined
 
 bin_PROGRAMS = openipmish
 
diff --git a/unix/Makefile.am b/unix/Makefile.am
index 5128839dad13..bd0f897ba8d6 100644
--- a/unix/Makefile.am
+++ b/unix/Makefile.am
@@ -10,12 +10,12 @@ lib_LTLIBRARIES = libOpenIPMIposix.la libOpenIPMIpthread.la
 libOpenIPMIpthread_la_LIBADD = -lpthread $(GDBM_LIB) \
 	$(top_builddir)/utils/libOpenIPMIutils.la $(RT_LIB)
 libOpenIPMIpthread_la_LDFLAGS = -rdynamic -version-info $(LD_VERSION) \
-	-L$(libdir) -no-undefined
+	-no-undefined
 
 libOpenIPMIposix_la_SOURCES = posix_os_hnd.c selector.c
 libOpenIPMIposix_la_LIBADD = $(top_builddir)/utils/libOpenIPMIutils.la \
 	$(GDBM_LIB) $(RT_LIB)
 libOpenIPMIposix_la_LDFLAGS = -rdynamic -version-info $(LD_VERSION) \
-	-L$(libdir) -no-undefined
+	-no-undefined
 
 noinst_HEADERS = heap.h
 
